### PR TITLE
[RHACS] Minor updates to 3.71 RN

### PR DESCRIPTION
**Pull request by kcarmichael08**
Version: `rhacs-docs-3.71`
Label: `RHACS/next`
Issue: none
[Preview](https://openshift-docs-81k40rfs9-kcarmichael08.vercel.app/openshift-acs/master/release_notes/371-release-notes.html)

### CODE REVIEW

**Review by tmalove — COMMENTED** [review]
LGTM

File: release_notes/371-release-notes.adoc [hunk 1]
```diff
@@ -70,9 +70,9 @@ Individual widget customization settings are short lived and are reset to the sy
...
+* *Images at most risk*: This widget lists the top six vulnerable images within the Dashboard-filtered scope, sorted by their computed risk priority, along with the number of critical and important CVEs they contain. Click an image name to go directly to the *Image Findings* page under *Vulnerability Management*. Use the *Options* menu to focus on fixable CVEs, or further focus on active images. *Note*: When clusters or namespaces have been selected in the Dashboard filter, the data displayed is already filtered to active images, or images that are being used by deployments within the filtered scope.
```
Comment (kalexand-rh) [inline release_notes/371-release-notes.adoc:75]: I suggest that you either remove `*Note*:` or format the note like a note: 

```suggestion
* *Images at most risk*: This widget lists the top six vulnerable images within the Dashboard-filtered scope, sorted by their computed risk priority, along with the number of critical and important CVEs they contain. Click an image name to go directly to the *Image Findings* page under *Vulnerability Management*. Use the *Options* menu to focus on fixable CVEs or further focus on active images.
+
[NOTE]
====
When clusters or namespaces are selected in the Dashboard filter, the data displayed is already filtered to active images or images that are being used by deployments within the filtered scope. 
====
```

Comment (kcarmichael08) [reply release_notes/371-release-notes.adoc:75]: TIL that you can keep the Note format under a bullet - thank you! Fixing now.

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**